packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -2386,14 +2385,14 @@ test('director notifiers', async t => {
     'Chit',
     defaultParamValues(chit.brand),
   );
-  state = await E(metrics).getUpdateSince(state.updateCount);
-  t.deepEqual(state.value, {
-    collaterals: [aethBrand, chit.brand],
-    rewardPoolAllocation: {},
+  await m.assertChange({
+    collaterals: { 1: chit.brand },
   });
 
   // Not testing rewardPoolAllocation contents because those are simply those values.
   // We could refactor the tests of those allocations to use the data now exposed by a notifier.
+
+  t.pass();
```

Comment:
I'd prefer to pass t into `metricsTracker()`, and have it use `t.is()` rather than `assert`.